src/ensembl/production/metadata/updater/core.py

```diff
@@ -91,40 +91,49 @@ def process_species(self, species_id, metadata_uri, taxonomy_uri, db_uri):
             genebuild_release_status = conn.check_release_status(self.genebuild_dataset.dataset_uuid)
 
             if organism_status == "New":
-                print ("New organism")
+                print("New organism")
```

Comment:
Are those `print` statement something for logging? If so, please use a logger. * which in the end should be essential part of figuring out what could go wrong in a loading.